frontend/src/employee-frontend/components/holiday-periods/FixedPeriodQuestionnaireForm.tsx

```diff
@@ -484,6 +549,47 @@ export default React.memo(function FixedPeriodQuestionnaireForm({
           )}
           hideErrorsBeforeTouched={hideErrorsBeforeTouched}
         />
+
+        <Label>{i18n.holidayQuestionnaires.conditionContinuousPlacement}</Label>
+        <FixedSpaceRow alignItems="center">
+          <DatePicker
+            info={useMemo(
+              () => errorToInputInfo(errors.start, i18n.validationErrors),
+              [errors.start, i18n.validationErrors]
+            )}
```

Comment:
Both of error conditions are wrong (`errors.start` and `errors.end`)

Reply:
Fixed 👍